cmd/probe/cli/switchover.go

```
@@ -0,0 +1,89 @@
+/*
+Copyright (C) 2022-2023 ApeCloud Co., Ltd
+
+This file is part of KubeBlocks project
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package cli
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	primary        string
+	candidate      string
+	sqlchannelAddr string
+)
+
+var SwitchCmd = &cobra.Command{
+	Use:   "switchover",
+	Short: "execute a switchover request.",
+	Example: `
+sqlctl switchover  --primary xxx --candidate xxx
+  `,
+	Args: cobra.MinimumNArgs(0),
+	Run: func(cmd *cobra.Command, args []string) {
+		var characterType string
+		if characterType = os.Getenv("KB_SERVICE_CHARACTER_TYPE"); characterType == "" {
+			characterType = "mysql"
```

it charType is not set, shall we raise an error?